### ISSUE TITLE
v0.4.0 hex release?

### ISSUE THREAD

**Issue by amokan** [connorjacobsen/tiktoken-elixir#7]
Any plans for a hex release of v0.4.0? Still showing v0.3.2 there.

Appreciate the library.

Thank you!

**Comment by arjan** [comment]
Yes that would be very nice! 👍🏻

**Comment by connorjacobsen** [comment]
Oops. Can do right now. Sorry about that.

**Comment by connorjacobsen** [comment]
Should be live! Sorry, dunno how I forgot to do that originally.

**Comment by arjan** [comment]
Looks like the `checksum-Elixir.Tiktoken.Native.exs` file was not updated before the release, the precompiled NIF fails to download:

```
==> tiktoken
Compiling 8 files (.ex)

== Compilation error in file lib/tiktoken/native.ex ==
** (RuntimeError) Error while downloading precompiled NIF: the precompiled NIF file does not exist in the checksum file. Please consider run: `mix rustler_precompiled.download Tiktoken.Native --only-local` to generate the checksum file..
```

**Comment by connorjacobsen** [comment]
My bad. Waiting on the GitHub actions to finish for v0.4.1 to fix it.

**Comment by connorjacobsen** [comment]
Okay should be fixed on `0.4.1`. Sorry about that.
